src/server.js

```diff
@@ -128,6 +128,13 @@ class ExecSession {
         this.execStream.end();
         break;
 
+      case msgcode.resize:
+        this.exec.resize({
+          h: message.readUInt16LE(1),
+          w: message.readUInt16LE(3),
+        });
+        break;
```

Comment:
Docker api says it's only valid if `tty` was given to create and start...

We should test what happens when this is violated... Probably best to ignore these messages if not started with tty...